lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -459,6 +459,40 @@ function dosomething_user_user_pass_submit($form, &$form_state) {
 }
 
 /**
+ * Trigger events after a user successfully logs in.
+ *
+ * Implements hook_user_login.
+ */
+function dosomething_user_user_login(&$form_state, $account) {
+  if (!module_exists('dosomething_northstar')) return;
+
+  $northstar_user = null;
+
+  // Figure out whether user is logging in via email or mobile. Then, we'll
+  // send a request to Northstar's `user/verify` endpoint to see if Northstar
+  // is also able to successfully authenticate with the given credentials.
+  if ($account = dosomething_user_get_user_by_email($form_state['values']['name'])) {
```

Comment:
I based this logic off [`dosomething_user_login_validate`](https://github.com/DFurnes/phoenix/blob/627a5b5e8bae0760a7ec19fde1fa228e882671da/lib/modules/dosomething/dosomething_user/dosomething_user.module#L907-L919) but let me know if there's a smarter way of handling it!